apps/website/src/components/Placeholder.astro

```diff
@@ -0,0 +1,19 @@
+---
+import { SvgInfoCircular } from '@itwin/itwinui-icons-react';
+---
+
+<div class='iui-alert' data-iui-status='informational'>
+  <SvgInfoCircular className='iui-alert-icon' aria-hidden='true' />
+  <span class='iui-alert-message'
+    >Some pages have not yet been finished and are being worked on. In the meantime, visit <a
+      class='iui-alert-link'
+      href='https://itwin.github.io/iTwinUI/react/?path=/story/overview--page'>storybook</a
```

Comment:
It would be cool if we routed users to same component in storybook. Can be added later.

Reply:
Updated